### ISSUE TITLE
Copy II+ V5.5 disk drive detection?

### ISSUE THREAD

**Issue by Keatah** [AppleWin/AppleWin#322]
In Copy II+ V5.5, on a real Apple II the program seems to detect whether you have one or two drives and sets its settings accordingly. By that I mean populating the status bar at the bottom of the screen.

In AppleWin it detects only Drive 1 unless you populate Drive 2 with an image prior to Copy II+ starting. Once the program is started, inserting a disk image does nothing unless you manually tell Copy II+ you have a 2nd drive.

Hope that makes sense.

**Comment by Michaelangel007** [comment]
Anyone have details on which I/O lines it using?  Q6L, Q6H, Q7L, Q7H ?

What is the relevant code in Copy ][+ v5.5 ?

**Comment by tomcw** [comment]
Tested with AppleWin 1.25.0.4 (authentic disk).
Still occurs with AppleWin 1.27.13.0 (both authentic & enhanced disk).

Disk-B is slot-6, drive-1 (see below). Whereas if you have a disk in drive-2 then Disk-B will be slot-6, drive-2.

```
           COPY ][ PLUS  5.5            
(C) 1982-5 CENTRAL POINT SOFTWARE, INC. 
----------------------------------------
                                        
COPY                  USE ARROW KEYS    
CATALOG DISK          & [RETURN] TO     
DELETE                SELECT FUNCTION   
LOCK/UNLOCK FILES                       
RENAME FILES                            
ALPHABETIZE CATALOG                     
FORMAT DISK                             
VERIFY                                  
VIEW FILES                              
TRACK/SECTOR MAP                        
SECTOR EDITOR                           
FIX FILE SIZES                          
CHANGE BOOT PROGRAM                     
UNDELETE FILES                          
NEW DISK INFO                           
QUIT                                    
                                        
DISK SLOT DRIVE DOS FREE&USED    PRINTER
  A    6    1   3.3                OFF  
  B    6    1   3.3                     
```

**Comment by tomcw** [comment]
This got fixed at some point between 1.27.13.0 and 1.30.6.0.
The following is just with "Copy II Plus 5.5.dsk" in drive 1; and drive 2 empty (demonstrating it's now working):

```
           COPY ][ PLUS  5.5            
(C) 1982-5 CENTRAL POINT SOFTWARE, INC. 
----------------------------------------
                                        
COPY                  USE ARROW KEYS    
CATALOG DISK          & [RETURN] TO     
DELETE                SELECT FUNCTION   
LOCK/UNLOCK FILES                       
RENAME FILES                            
ALPHABETIZE CATALOG                     
FORMAT DISK                             
VERIFY                                  
VIEW FILES                              
TRACK/SECTOR MAP                        
SECTOR EDITOR                           
FIX FILE SIZES                          
CHANGE BOOT PROGRAM                     
UNDELETE FILES                          
NEW DISK INFO                           
QUIT                                    
                                        
DISK SLOT DRIVE DOS FREE&USED    PRINTER
  A    6    1   3.3                OFF  
  B    6    2   3.3                     
```

Probably fixed at #748 (ie. AppleWin 1.29.8.0).
